### PR TITLE
add  maintainer notes for ffi

### PR DESCRIPTION
**Pull request by jennijuju**

### CODE REVIEW

**Comment by jennijuju** [comment]
@Stebalien @cryptonemo I forgot in our meeting, did we discuss:
- when its time to do a lotus mandatory release, say lotus `release/v1.18.0`
- will all consensus breaking changes in ffi first to be merged into a `next` branch, integrated into lotus, being tested then merge to master?
- or all consensus breaking `feat/` branches should be merged into master, we create a branch base of master as `release/lotus-v1.18.0`, and then introduce it to lotus for testing?

I think the 2nd option makes more sense, tho con of that is:
- when lotus has a feature release & mandatory release going on at the same time, it gonna be tricky to manage the back porting ?

**Comment by Stebalien** [comment]
IIRC:

- next: consensus breaking changes, merged to next when the upgrade is scheduled/finalized.
- master: all other changes.
- release/v: backports/patches.

For mandatory releases with consensus breaking changes..., I think it'll depend on the changes.

- I expect we'll usually just use FFI master.
- If FFI master has complex changes that aren't entangled with the network upgrade, consider a backport.

_Ideally_, we'd cut a feature release a month or so before the mandatory release (IMO).

**Review by Stebalien — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -94,6 +94,38 @@ Run it like so:
 ./mkreleaselog v0.25.0 v0.26.0 > /tmp/v0.26.0.notes.txt
 ```
 
+## Contribution 
+
+### Maintainers
+
+The core maintainers of this repository are:
+- @nemo, @vmx & @shawnrader, from the fil-crypto team
```
Comment (vmx) [inline README.md:102]: I'm not really a maintainer. I surely help out, but I think it should be @nemo and @shawnrader only.

**Review by vmx — COMMENTED** [review]
I'm not deeply involved in filecoin-ffi maintenance. I'd refer to @shawnrader for an approval for the FilCrypto team.

---

There are typos, all "it's" should be "its".

**Review by arajasek — APPROVED** [review]
This SGWM

**Pull request merged**